### DISCUSSION TITLE
Possible to change the system variable "Anwensenheit"/"Presence" via HA?

### DISCUSSION THREAD

**Discussion by TheUntouchable** [SukramJ/aiohomematic#1353] [Q&A] [answered]
Hello folks!

I'm currently experimenting a bit with HA and the possibilities.

At the moment I'm changing my presence in HM and in my Synology Surviliance Station with Tasker from my phone via APIs.

For the SSS I have already managed to change this via HA, but not for HM.

I can only see the status of the system variable, but not change the value, and I had to activate it first in the integration.

Is there a way to change this via HA as well?

Many thanks in advance!

**Comment by SukramJ** [comment, marked as answer]
Yes, it is possible.
I always recommend reading the [docs](https://github.com/danielperna84/custom_homematic#homematicip-local-documentation).
There is an extra chapter to [extend the sysvar](https://github.com/danielperna84/custom_homematic#system-variables) support.

**Comment by TheUntouchable** [comment]
Thanks, that was quite easy to add "hahm" to the variable description :)
